### DISCUSSION TITLE
Free zoom with mouse wheel?

### DISCUSSION THREAD

**Discussion by Ander3232** [sumatrapdfreader/sumatrapdf#3141] [Ideas]
When you hold Ctrl and use the mouse to zoom in/out, is it normal for SPDF to jump incrementally to the pre-set zoom levels on the Zoom menu? That's what it's doing for me, anyway (e.g. jumping from 100% to 125%, with no choices between.)

If this is how SPDF actually operates (and not just a quirk of my system), will you please consider allowing us to zoom in/out freely using the mouse wheel? It would make using the app so much easier—being able to zoom to the desired size without having to fuss with the "Custom Zoom" box every time. Thanks!

**Comment by Nac110** [comment]
In `Settings > Advanced Options...` you can change the zoom options.

What you are looking for is `ZoomIncrement`, which will change zoom by the defined increment number. 
It is by default 0, which only uses `ZoomLevels` zoom presets.

**Reply by Ander3232** [reply]
Sorry, just seeing your reply now (remodeling chaos!). This is perfect—thanks very much for pointing it out. Cheers!

**Comment by GitHubRulesOK** [comment]
@Ander3232 for wider discussion of @Nac110 s comment see https://github.com/sumatrapdfreader/sumatrapdf/issues/2270
